src/main/signal_handlers.cpp

```diff
@@ -21,6 +21,7 @@
 #include <string.h>
 
 #include <cerrno>
+#include <chrono>
```

Comment:
This include shouldn't be needed, right?

Reply:
You are right. I removed it.